### ISSUE TITLE
samsung browser needs to use body, not html

### ISSUE THREAD

**Issue by makc** [cmswalker/fullpage-react#85]
setting scrollTop on html has no effect there, so I had to put following code into determineVerticalRoot:
```javascript
    // http://developer.samsung.com/technical-doc/view.do?v=T000000203
    if (/SAMSUNG.*Build\//.test(userAgent)) {
      return document.body;
    }
```

**Comment by makc** [comment]
btw, you'll need some polyfills for samsung browser to even beging working, e g 
* https://cdnjs.cloudflare.com/ajax/libs/es6-shim/0.34.2/es6-shim.min.js
* https://www.promisejs.org/polyfills/promise-6.1.0.js

**Comment by cmswalker** [comment]
Thanks again. Releasing this now, polyfills should be up to the consumer.

**Comment by cmswalker** [comment]
fixed in 3.0.32
